### ISSUE TITLE
2D Labels

### ISSUE THREAD

**Issue by YYYYzhuo** [Soptq/e-3sfc#2]
In compute_synthetic_sample function, the variable synthetic_labels is two dimensional, Why can we use 2D variables as parameters of the cross entropy loss function? I can't figure it out.
